string V1
string V2
string V3
string V4
string C1
string C2
string C3
string C4
string THRUST1
string STRAIN1
string THRUST2
string STRAIN2
string THRUST3
string STRAIN3
string THRUST4
string STRAIN4
string RPM1
string RPM2
string RPM3
string RPM4
